docs/devel/networking.md

```diff
@@ -72,6 +72,74 @@ As of now, the existent binding mechanisms are:
 - [slirp](#slirp-binding-mechanism)
 
 ### Bridge binding mechanism
+Using the bridge `BindMechanism` requires a VMI configuration featuring a
+network whose interface type is `bridge` - the yaml file below can be used
+as reference, but please refer to the
+[user guide](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=bridge)
+for more information.
+```yaml
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: default
+          bridge: {}
+  networks:
+  - name: default
+    pod: {} # Stock pod network
+```
+
+Let's refer to the image below to get a better understanding of how this
+`BindMechanism` works.
+
+Bridge binding mechanism diagram
+
+![alt text](https://github.com/kubevirt/kubevirt.github.io/blob/source/assets/images/diagram.png)
+
+As can be seen in the diagram above, there are three actors at play: CNI,
+libvirt, and DHCP. For completeness sake, let's add one more actor that is
+implicit in the picture: KubeVirt.
+
+Once the bridge `BindMechanism` is triggered,
+[CNI](https://github.com/containernetworking/plugins/tree/master/plugins/main/bridge)
+will create / configure a linux bridge in the k8s node. It will also create a
+veth pair, attaching one end of it to the bridge, and move the other end into
+the pod.
+
+KubeVirt, in the
+[preparePodNetworkInterfaces method](https://github.com/kubevirt/kubevirt/blob/5995d2164faefdcc7a59a857e12f8ed4cdb7e094/pkg/virt-launcher/virtwrap/network/podinterface.go#L323),
+updates the in-pod veth mac-address, creates an in-pod bridge, and connects
+them. This happens in the phase#1 - i.e. performed by virt-handler on behalf
+of virt-launcher.
+
+In phase#2 (executed by virt-launcher, unprivileged) KubeVirt also
+participates; in the
+[Convert_v1_VirtualMachine_To_api_Domain](https://github.com/kubevirt/kubevirt/blob/ab3f1ff983b72e3e97863b12e23d580d60025592/pkg/virt-launcher/virtwrap/api/converter.go#L1208)
+function, it will generate an element of the domain xml instructing the libvirt
+to connect to the in-pod linux bridge. It looks like:
+
+```xml
+ <interface type='bridge'>
+    <source bridge='k6t-eth0'/>
+    <model type='virtio'/>
+ </interface>
+```
+
+Furthermore, in the
+[decorateConfig](https://github.com/kubevirt/kubevirt/blob/5995d2164faefdcc7a59a857e12f8ed4cdb7e094/pkg/virt-launcher/virtwrap/network/podinterface.go#L364)
+method, KubeVirt will also configure the MTU, and optionally the MAC
+address, of the link libvirt will create.
```

Comment:
Seems like privileged actions. Is this back to phase#1?

Reply:
Nope, it's just grabbing those values from the pod interface and copying them to the interface dom xml. 

Is it clearer now ?